cli/cmd/get.go

```diff
@@ -361,7 +375,11 @@ func regressionMetricsTable(apiMetrics *schema.APIMetrics) string {
 }
 
 func classificationMetricsTable(apiMetrics *schema.APIMetrics) string {
-	classList := make([]string, len(apiMetrics.ClassDistribution))
+	numClasses := 0
+	if apiMetrics != nil {
+		numClasses = len(apiMetrics.ClassDistribution)
+	}
+	classList := make([]string, numClasses)
 
 	i := 0
 	for inputName := range apiMetrics.ClassDistribution {
```

Comment:
Will this be a nil pointer if `apiMetrics` is nil? (same question for other references to `apiMetrics` in this function)